<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Princeton plainsboro</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonClinicInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Информация
о поликлинике</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="accountPages">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>551</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="personalAccount">
     <widget class="QLabel" name="labelEmail">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="personalPhoto">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>201</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelPhone">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelName">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>191</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="buttomChangePhoto">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Изменить фото</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>180</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Редактировать
профиль</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPasswordEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Изменить
пароль</string>
      </property>
     </widget>
     <widget class="QTableView" name="viewTalon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>511</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRemoveTalon">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>400</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Телефон:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>e-mail:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QComboBox" name="comboBoxDepartment">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>149</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберете отделение:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxService">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>122</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберете услугу:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSpecialist">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>164</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберете специалиста:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSetTalon">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>390</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Оформить талон</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDataSelect">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>105</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Выбрать дату</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editDateTime">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTimeEdit" name="editTime">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="maximumTime">
       <time>
        <hour>16</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="minimumTime">
       <time>
        <hour>8</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="displayFormat">
       <string>H:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonAccount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Личный аккаунт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonService">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Запись на услугу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
